user_data/strategies/util.py

```diff
@@ -1,21 +1,25 @@
 import subprocess
 import threading
+from user_data.strategies.config import Config
 
-BACKTEST_DOWNLOADED_JSON_DATA_FILE_PATH = ""
-BACKTEST_YEAR = 2020
-BACKTEST_MONTH_INDEX = 9
-IS_BACKTEST = False
-WORKSPACE_PATH = "workspace2" if IS_BACKTEST else "workspace"
-EXECUTION_PATH = "/root/" + WORKSPACE_PATH + "/execution/"
-
-
-def launcher(mode, coin, brain):
-    threading.Thread(target=_perform_launcher, args=(mode, coin, brain)).start()
+def launcher(mode, coin, brain,date_time = 12344315):
+    if Config.IS_BACKTEST:
```

Comment:
this is wrong, backtest shouldn't point to _perform_launcher

create _perform_backtest()